com.unity.render-pipelines.core/Editor/Camera/CameraUI.Drawers.cs

```diff
@@ -75,89 +68,117 @@ public static void Drawer_Projection(ISerializedCamera p, Editor owner)
             }
             EditorGUI.EndProperty();
 
-            if (cam.orthographic.hasMultipleDifferentValues)
-                return;
+            return projectionType;
+        }
 
-            if (projectionType == ProjectionType.Orthographic)
+        static void DrawerOrthographicType(ISerializedCamera p, Editor owner)
+        {
+            EditorGUILayout.PropertyField(p.baseCameraSettings.orthographicSize, Styles.sizeContent);
+            Drawer_FieldClippingPlanes(p, owner);
+        }
+
+        static void DrawerPerspectiveType(ISerializedCamera p, Editor owner)
+        {
+            var cam = p.baseCameraSettings;
+
+            float fovCurrentValue;
+            bool multipleDifferentFovValues = false;
+            bool isPhysicalCamera = p.projectionMatrixMode.intValue == (int)ProjectionMatrixMode.PhysicalPropertiesBased;
+
+            var rect = EditorGUILayout.GetControlRect();
+
+            var guiContent = EditorGUI.BeginProperty(rect, Styles.FOVAxisModeContent, cam.fovAxisMode);
+            EditorGUI.showMixedValue = cam.fovAxisMode.hasMultipleDifferentValues;
+
+            EditorGUI.BeginChangeCheck();
+            var fovAxisNewVal = (int)(Camera.FieldOfViewAxis)EditorGUI.EnumPopup(rect, guiContent, (Camera.FieldOfViewAxis)cam.fovAxisMode.intValue);
+            if (EditorGUI.EndChangeCheck())
+                cam.fovAxisMode.intValue = fovAxisNewVal;
```

Comment:
Just a comment:
I remember having done an utility method to handle enum in SerializedProperty (get and set). Here it is good but some times I see ones using index value. Perhaps we should move the utility in Core if not already done. It also auto cast the int value in the requested enum type.